FrontEnd/src/components/SignUp/components/signup/signup-form/RulesModal.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './RulesModal.css';
```

Comment:
On the project we use filename.module.css

Reply:
done